src/background/sessionBackground.ts

```diff
@@ -0,0 +1,48 @@
+import Background from '.';
+import { MESSAGE_TYPE, RESPONSE_TYPE } from '../constants';
+
+export default class SessionBackground {
+  private static SESSION_LOGOUT_INTERVAL_MS: number = 600000;
```

Comment:
remove a 0, should be `60,000ms = 60 sec`

Reply:
you only want it to last for 1 min before they have to log back in? seems pretty short to me, which is why I picked 10 minutes

Reply:
ya i thought 1 min was sufficient, but i guess we should add this as an option in the settings. can leave it 10 mins for now for dev purposes. but we can add a follow up settings page to set the timeout length.